<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);

</string>
  </property>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Partner</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Task name</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Task_browser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color:#FFF;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="description_browser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color:#FFF;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="partner_browser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color:#FFF;
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="location_browser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color:#FFF;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Stutes</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="stutes_browser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color:#FFF;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>End Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Start Time</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="start_browser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>570</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color:#FFF;
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="end_browser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>650</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color:#FFF;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Edit_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>740</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>740</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>740</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
